src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react';
+import Calculator from './components/Calculator';
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends Component {
+  render() {
+    return <Calculator />;
+  }
 }
```

Comment:
- [ ] **[OPTIONAL]** You can fix the Eslint error instead of disabling it by extending React.pureComponent or importing pureComponent instead of Component. Another way to fix this is by creating a constructor and a state.

Reply:
Thanks, that worked  👍